### PR TITLE
fix/qb1453: use pp local pingtime while checking latency

### PR DESCRIPTION
**Pull request by bjlqsnetwork**
- qb1453: use pp local pingtime while checking latency

### CODE REVIEW

File: pp/peers/sp_peers.go [hunk 1]
```diff
@@ -23,6 +23,8 @@ import (
 
 var bufferedSpConns = make([]*cf.ClientConn, 0)
 
+var PingTimeSPMap = make(map[string]int64, 0)
```
Comment (alexstratos) [inline pp/peers/sp_peers.go:26]: this might have concurrency issue

File: pp/peers/pp_list.go [hunk 2]
```diff
@@ -20,6 +19,8 @@ import (
 // PeerList is a list of the know PP node peers
 var peerList types.PeerList
 
+var PingTimePPMap = make(map[string]int64, 0)
```
Comment (alexstratos) [inline pp/peers/pp_list.go:22]: this might have concurrency issue

File: pp/peers/sp_peers.go (hunk 1 above)
Comment (bjlqsnetwork) [reply pp/peers/sp_peers.go]: changed to sync.map, testing ongoing

File: pp/peers/pp_list.go (hunk 2 above)
Comment (bjlqsnetwork) [reply pp/peers/pp_list.go]: changed to sync.map, testing ongoing

**Comment by bjlqsnetwork** [comment]
@alexstratos test ok

**Review by alexstratos — APPROVED** [review]

**Pull request merged**